Sources/StreamChatUI/Common Views/BaseViews.swift

```diff
@@ -50,8 +50,8 @@ open class View: UIView, AppearanceSetting, Customizable {
         guard superview != nil else { return }
         
         setUp()
-        (self as! Self).applyDefaultAppearance()
         setUpAppearance()
+        (self as! Self).applyDefaultAppearance()
         setUpLayout()
         updateContent()
```

Comment:
This is not the order I was suggesting in the comment 🙃

Reply:
Edited the commit, now it should be 😉